src/includes/sourcemaps-wizard-instructions.mdx

```diff
@@ -0,0 +1,17 @@
+```bash
+npx @sentry/wizard@latest -i sourcemaps
+```
+
+The wizard will guide you through the following steps:
+
+- Connecting your local project to Sentry
+- Installing the necessary Sentry packages
+- Configuring your build tool to upload source maps
+- Configuring your CI to upload source maps in CI
+
+<Alert level="warning">
+
+Configuring source maps upload with the Sentry Wizard is still experimental and [under development](https://github.com/getsentry/sentry-wizard/issues/290).
+Let us know on [GitHub](https://github.com/getsentry/sentry-wizard/issues/new/choose) if you have feedback or issues with the wizard.
```

Comment:
```suggestion
Let us know on [GitHub](https://github.com/getsentry/sentry-wizard/issues/new/choose) if you have feedback or issues.
```

I think this reads better because we have a lot of "with the wizard"